### ISSUE TITLE
Upgrading to Orval 6.5 breaks generation

### ISSUE THREAD

**Issue by Primajin** [anymaniax/orval#301]
## What are the steps to reproduce this issue?
1. Install Orval 6.4 and generate
2. Upgrade to 6.5

## What happens?
The generation breaks

## What were you expecting to happen?
The generation works as before - e.g. no breaking change from 6.4 to 6.5

## Any logs, error output, etc?
```
🍻 Start orval v6.5.1 - A swagger client generator for typescript
 > request.ts:1:40: error: [plugin: replace-import-meta] ENOENT: no such file or directory, open 'C:\mypath\typesaxios.ts'
    1 │ import Axios, {AxiosRequestConfig} from 'axios';
      ╵                                         ~~~~~~~
   ../../node_modules/orval/dist/chunk-RFHR4BKT.js:4:5855: note: This error came from the "onLoad" callback registered here
    4 │ ...},{name:"replace-import-meta",setup(i){i.onLoad({filter:/\.[jt]s$/},async c=>{let p=awa...
      ╵                                             ~~~~~~
    at setup (C:\mypath\node_modules\orval\dist\chunk-RFHR4BKT.js:4:5856)
    at handlePlugins (C:\mypath\node_modules\orval\node_modules\esbuild\lib\main.js:756:23)
    at Object.buildOrServe (C:\mypath\node_modules\orval\node_modules\esbuild\lib\main.js:1044:7)
    at C:\mypath\node_modules\orval\node_modules\esbuild\lib\main.js:1895:17
    at new Promise (<anonymous>)
    at Object.build (C:\mypath\node_modules\orval\node_modules\esbuild\lib\main.js:1894:14)
    at build (C:\mypath\node_modules\orval\node_modules\esbuild\lib\main.js:1749:51)
    at Qi (C:\mypath\node_modules\orval\dist\chunk-RFHR4BKT.js:4:4794)
    at ut (C:\mypath\node_modules\orval\dist\chunk-RFHR4BKT.js:4:4497)
```
vs.
```
🍻 Start orval v6.4.3 - A swagger client generator for typescript
Your mutator cannot be loaded so default setup has been applied => C:/mypath/request.ts
🎉 browser - Your OpenAPI spec has been converted into ready to use orval!
```

## Any other comments?
…

## What versions are you using?
**Operating System:** Windows 10
**Package Version:** 6.5.1

**Comment by anymaniax** [comment]
Hey @Primajin, sorry for you. Can you provide the specification or a small example that reproduce the problem?

**Comment by Primajin** [comment]
Of course, will try to set up an example in the coming week. Do you have a preferred code sandbox?

**Comment by anymaniax** [comment]
No, you can take what you want. And thanks 😄

**Comment by Primajin** [comment]
OK I found the issue:
When using
```json
    "paths": {
      "*": ["something"],
    }
```
in the `tsconfig.json` - it breaks in `6.5` while it works in `6.4` (where something can be anything)

I set up: https://github.com/Primajin/orval-sandbox

- `6.4.3` works fine: https://github.com/Primajin/orval-sandbox/runs/4978704393?check_suite_focus=true#step:5:8
- `6.5.1` explodes: https://github.com/Primajin/orval-sandbox/runs/4978704391?check_suite_focus=true#step:5:8

**Comment by anymaniax** [comment]
fixed will be in the next version

**Comment by anymaniax** [comment]
fixed with `6.6.1`

**Comment by Primajin** [comment]
Great! I can confirm:

```
🍻 Start orval v6.6.1 - A swagger client generator for typescript
🎉 browser - Your OpenAPI spec has been converted into ready to use orval!
```
